scripts/generate.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Remove the existing 'seam' directory
+rm -rf ./seam
+
+# Generate SDK files
+nextlove-sdk-generator generate python ./temp_sdk
+
+# Move only the 'seam' folder
+mv ./temp_sdk/seam ./seam
+
+# Clean up the temporary SDK generation folder
+rm -rf ./temp_sdk
```

Comment:
```suggestion
# Remove the existing temporary SDK generation folder
rm -rf ./tmp/nextlove-sdk-generator-output

# Generate SDK files
nextlove-sdk-generator generate python ./tmp/nextlove-sdk-generator-output

# Move only the 'seam' folder
rm -rf ./seam
mv ./tmp/nextlove-sdk-generator-output ./seam

# Clean up the temporary SDK generation folder
rm -rf ./tmp/nextlove-sdk-generator-output
```

Reply:
Not sure I understand line 4 `rm -rf ./tmp/nextlove-sdk-generator-output` as temporary directory doesn't exist at this time it gets created by subsequent `nextlove-sdk-generator generate python ./tmp/nextlove-sdk-generator-output` command.

Reply:
I missed adding `mkdir -p`. 

```suggestion
mkdir -p ./tmp

# Remove the existing temporary SDK generation folder
rm -rf ./tmp/nextlove-sdk-generator-output

# Generate SDK files
nextlove-sdk-generator generate python ./tmp/nextlove-sdk-generator-output

# Move only the 'seam' folder
rm -rf ./seam
mv ./tmp/nextlove-sdk-generator-output ./seam

# Clean up the temporary SDK generation folder
rm -rf ./tmp/nextlove-sdk-generator-output
```

Reply:
> Not sure I understand line 5 `rm -rf ./tmp/nextlove-sdk-generator-output` as temporary directory doesn't exist at this time. It gets created by subsequent `nextlove-sdk-generator generate python ./tmp/nextlove-sdk-generator-output` command.

This ensures the script will always run in a clean state, even if it failed party way though before it was able to remove the tmp dir.

Reply:
https://github.com/seamapi/python-next/pull/6/commits/09eedb6ab0203feae7bb40fbc5f7573d9cf5a21f